### DISCUSSION TITLE
Call for Research Participants

### DISCUSSION THREAD

**Discussion by andyjih** [grouparoo/grouparoo#1278] [Chat]
Hello everyone! We're hoping to chat with people who have deployed Grouparoo to production. We're curious to chat more about your experience finding and using Grouparoo. 

If you're open to a short phone call (20 minutes), please email research@grouparoo.com. As thanks for your time, we'll provide a gift card to a vendor of your choosing. Thanks!

**Comment by evantahler** [comment]
(Locked the conversation because you should email us, rather than reply here!)
